components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Person from '@material-ui/icons/Person';
+import Icon from '../Icon/Icon';
+import { colors } from '../shared/theme';
+
+const borderStyle = 'border-radius: 50% !important;';
+
+const AvatarWrapper = styled.a`
+  display: inline-flex;
+  position: relative;
+  cursor: pointer;
+  height: 24px;
+  text-decoration: unset;
+  color: unset;
+`;
+
+const AvatarImage = styled.img.attrs({
+  alt: 'user picture',
+  ariaHidden: 'true',
+})`
+  ${borderStyle}
+  width: 24px;
+  height: 24px;
+`;
+
+const AvatarIcon = styled(Person).attrs({
+  style: {
+    color: colors.neutral[300],
+    fontSize: '18px',
+  },
+})`
+  ${borderStyle}
+  background: ${colors.neutral[700]};
+  padding: 3px;
+`;
+
+const AvatarNotification = styled.span`
+  ${borderStyle}
+  background-color: ${colors.secondary[700]};
+  position: absolute;
+  left: 16px;
```

Comment:
According to the style guide is 12px instead of 16px

Reply:
16 is the distance from the left

Reply:
My fault